<launch>
	<arg name="config_file" default="$(find rasberry_des)/config/polytunnel_des_config.yaml" />
	<arg name="map" default="$(find rasberry_des)/maps/empty_world.yaml" />

	<node pkg="tf" type="static_transform_publisher" name="static_transform_publisher" args="0 0 0 0 0 0 map odom 10"/>

	<node pkg="map_server" type="map_server" name="map_server" args="$(arg map)" />
	
	<rosparam command="load" file="$(arg config_file)" ns="/rasberry_des_config/"/>

</launch>
